examples/website/quick-start/package.json

```diff
@@ -9,8 +9,8 @@
     "build": "vite build"
   },
   "dependencies": {
-    "deck.gl": "^8.9",
-    "hubble.gl": "^1.4.0",
+    "deck.gl": "^9.1",
+    "hubble.gl": "^2.0.0-alpha.3",
```

Comment:
I recall deleting the QuickStart component (or at least planning to) in 2.0. Are you sure this works?

Reply:
Thanks for testing it. And me too, I don't want to remove the non-geo example but I question the need for Hubble to maintain the QuickStart component - it's a nice concept but in practice it doesn't seem very valuable since each project tends to set Deck up in their own way. 

I'd originally wrote the component because the setup was verbose and repetitive in the docs, but the better solution would be to make core changes to deck and Hubble so that the setup is more concise.